aten/src/ATen/native/vulkan/VulkanAten.cpp

```diff
@@ -259,6 +259,42 @@ at::Tensor vulkan_reshape(at::Tensor const& input, IntArrayRef shape) {
       input.options());
 }
 
+at::Tensor vulkan_cat(TensorList tensors, int64_t dim) {
```

Comment:
`TensorList` is an alias to `ArrayRef<Tensor>`, thats why reference to it will not work:
```
terminate called after throwing an instance of 'c10::Error'
  what():  *cpp_signature == *cpp_signature_ INTERNAL ASSERT FAILED at "../aten/src/ATen/core/dispatch/OperatorEntry.cpp":93, please report a bug to PyTorch. Tried to register a kernel (registered at ../aten/src/ATen/native/vulkan/VulkanAten.cpp:618) for operator aten::_cat for dispatch key Vulkan, but the C++ function signature at::Tensor (c10::ArrayRef<at::Tensor> const&, long) mismatched with a previous kernel that had the signature at::Tensor (c10::ArrayRef<at::Tensor>, long)
Exception raised from registerKernel at ../aten/src/ATen/core/dispatch/OperatorEntry.cpp:93 (most recent call first):
frame #0: c10::Error::Error(c10::SourceLocation, std::string) + 0x4d (0x7f155c08477d in /home/ivankobzarev/pytorch-vulkan/pytorch/build/lib/libc10.so)
```
But const works , I will add it